llvm/utils/repo/Makefile

```diff
@@ -16,8 +16,12 @@ else
 endif
 # The path of the llvm-project-prepo build directory.
 LLVM_BUILD = ${LLVM_REPO}/build
-# The path of the directory in which llvm libraries are installed.
+# The path of the directory in which llvm libraries (except libcompiler-rt.a) are installed.
 LLVM = ${HOME}/LLVM
+# The LLVM version. By default. it is 11.0.0.

```

Comment:
I'm not sure that this comment really tells us anything!

Reply:
I have updated the comment in https://github.com/SNSystems/llvm-project-prepo/commit/cdde1843a6d666706b13b94d9d7b489876238ac1.